src/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import LoginForm from "../components/Formulario-login/LoginForm";
+
+const AppRoutes: React.FC = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<LoginForm />} />
```

Comment:
TO-DO: El path `/` mostrará un componente que use el Hero 3D, la ruta `/login` tendrá el componente `LoginForm`